<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1062</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>50</y>
      <width>1061</width>
      <height>531</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="showGrid">
     <bool>false</bool>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="sortingEnabled">
     <bool>false</bool>
    </property>
    <property name="cornerButtonEnabled">
     <bool>false</bool>
    </property>
    <property name="columnCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>-10</y>
      <width>601</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QRadioButton" name="radioButton">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Dating</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Biznis</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_3">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Prijatelji</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_4">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Rodbina</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Svi</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QLineEdit" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Ukucajte ime ili prezime...</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>10</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1062</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFajl">
    <property name="title">
     <string>Fajl</string>
    </property>
    <addaction name="actionNovi"/>
    <addaction name="actionUvoz"/>
    <addaction name="actionIzvoz"/>
    <addaction name="separator"/>
    <addaction name="actionIzadi"/>
   </widget>
   <widget class="QMenu" name="menuIzmjena">
    <property name="title">
     <string>Izmjena</string>
    </property>
    <addaction name="actionOsvjezi"/>
    <addaction name="separator"/>
    <addaction name="actionDodaj_novi_kontakt"/>
    <addaction name="actionUredi_kontakt"/>
    <addaction name="actionObrisi_kontakt"/>
   </widget>
   <widget class="QMenu" name="menuInformacije">
    <property name="title">
     <string>Informacije</string>
    </property>
    <addaction name="actionPatch_notes"/>
    <addaction name="actionQt"/>
   </widget>
   <addaction name="menuFajl"/>
   <addaction name="menuIzmjena"/>
   <addaction name="menuInformacije"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNovi"/>
   <addaction name="separator"/>
   <addaction name="actionDodaj_novi_kontakt"/>
   <addaction name="actionUredi_kontakt"/>
   <addaction name="actionObrisi_kontakt"/>
   <addaction name="separator"/>
   <addaction name="actionOsvjezi"/>
  </widget>
  <action name="actionDodaj_novi_kontakt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/Icons/business_application_addmale_useradd_insert_add_user_client_2312.ico</normaloff>:/images/Icons/business_application_addmale_useradd_insert_add_user_client_2312.ico</iconset>
   </property>
   <property name="text">
    <string>Dodaj novi kontakt</string>
   </property>
  </action>
  <action name="actionUredi_kontakt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/Icons/businessapplication_edit_male_user_thepencil_theclient_negocio_2321.ico</normaloff>:/images/Icons/businessapplication_edit_male_user_thepencil_theclient_negocio_2321.ico</iconset>
   </property>
   <property name="text">
    <string>Uredi kontakt</string>
   </property>
  </action>
  <action name="actionObrisi_kontakt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/Icons/delete_delete_deleteusers_delete_male_user_maleclient_2348.ico</normaloff>:/images/Icons/delete_delete_deleteusers_delete_male_user_maleclient_2348.ico</iconset>
   </property>
   <property name="text">
    <string>Obriši kontakt</string>
   </property>
  </action>
  <action name="actionOsvjezi">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/Icons/Refresh_36729.ico</normaloff>:/images/Icons/Refresh_36729.ico</iconset>
   </property>
   <property name="text">
    <string>Osvježi</string>
   </property>
  </action>
  <action name="actionPatch_notes">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/Icons/ballpoint_pen_stationery_notes_paper_sticky_note_icon_256745.ico</normaloff>:/images/Icons/ballpoint_pen_stationery_notes_paper_sticky_note_icon_256745.ico</iconset>
   </property>
   <property name="text">
    <string>DSContact</string>
   </property>
  </action>
  <action name="actionQt">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/Icons/qt_94938.ico</normaloff>:/images/Icons/qt_94938.ico</iconset>
   </property>
   <property name="text">
    <string>Qt</string>
   </property>
  </action>
  <action name="actionUvoz">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/Icons/import_icon-icons.com_52387.ico</normaloff>:/images/Icons/import_icon-icons.com_52387.ico</iconset>
   </property>
   <property name="text">
    <string>Učitaj</string>
   </property>
  </action>
  <action name="actionIzvoz">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/Icons/export_icon-icons.com_52383.ico</normaloff>:/images/Icons/export_icon-icons.com_52383.ico</iconset>
   </property>
   <property name="text">
    <string>Sačuvaj</string>
   </property>
  </action>
  <action name="actionNovi">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/Icons/new-file_40454.ico</normaloff>:/images/Icons/new-file_40454.ico</iconset>
   </property>
   <property name="text">
    <string>Novi</string>
   </property>
  </action>
  <action name="actionIzadi">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/images/Icons/systemshutdown_103390.ico</normaloff>:/images/Icons/systemshutdown_103390.ico</iconset>
   </property>
   <property name="text">
    <string>Izađi</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
